sql/2020/07_SEO/10_07b_title_tag_length_percentiles_raw.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# 10_07b: <title> length
+SELECT
+  percentile,
+  client,
+  APPROX_QUANTILES(LENGTH(title), 1000)[OFFSET(percentile * 10)] AS title_length
+FROM (
+  SELECT
+    client,
+    REGEXP_EXTRACT(body, '(?i)<title>([^(</title>)]*)</title>') AS title
```

Comment:
Proposed custom metric:

```js
document.title
```

Reply:
I'm using a selector as we SEOs want to know if they add multiple titles.